Homework_6/blocks/Modal.css

```diff
@@ -0,0 +1,33 @@
+.overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+
+    width: 100vw;
+    height: 100vh;
+
+    z-index: 1000;
+
+    background: rgba(0, 0, 0, 0.7);
+}
+
+.modal {
+    position: absolute;
```

Comment:
Seems like `position: absolute` is redundant and you can position this modal using css styles in `overlay` component